### PR TITLE
Update targets supported by Polly

### PR DESCRIPTION
**Pull request by reisenberger**
## Summary

Updates the target frameworks quoted as supported by Polly, to reflect the latest 

Reference: https://github.com/App-vNext/Polly/wiki/Supported-targets

(Contributor reference: I'm the lead Polly maintainer.)

### CODE REVIEW

File: docs/architecture/microservices/implement-resilient-applications/implement-http-call-retries-exponential-backoff-polly.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ ms.date: 01/07/2019
 
 The recommended approach for retries with exponential backoff is to take advantage of more advanced .NET libraries like the open-source [Polly library](https://github.com/App-vNext/Polly).
 
-Polly is a .NET library that provides resilience and transient-fault handling capabilities. You can implement those capabilities by applying Polly policies such as Retry, Circuit Breaker, Bulkhead Isolation, Timeout, and Fallback. Polly targets .NET 4.x and the .NET Standard Library 1.0 (which supports .NET Core).
+Polly is a .NET library that provides resilience and transient-fault handling capabilities. You can implement those capabilities by applying Polly policies such as Retry, Circuit Breaker, Bulkhead Isolation, Timeout, and Fallback. Polly targets .NET 4.x and the .NET Standard Library 1.1 and 2.0 (which supports .NET Core).
```
Comment (mairaw) [inline docs/architecture/microservices/implement-resilient-applications/implement-http-call-retries-exponential-backoff-polly.md]:
```suggestion
Polly is a .NET library that provides resilience and transient-fault handling capabilities. You can implement those capabilities by applying Polly policies such as Retry, Circuit Breaker, Bulkhead Isolation, Timeout, and Fallback. Polly targets .NET Framework 4.x and the .NET Standard 1.1 and 2.0 (which supports .NET Core).
```

**Review by mairaw — CHANGES_REQUESTED** [review]
Thanks @reisenberger. I left one suggestion unrelated to your change to fix some branding issues but I also have some questions. Should we mention .NET Standard support is from 1.0 through 2.0 or do you prefer to cover only current? Should we also mention 3.5 support?

**Comment by reisenberger** [comment]
Thanks for the good questions around supported Polly targets. 

>Should we mention .NET Standard support is from 1.0 through 2.0 or do you prefer to cover only current? 

I pushed a commit to incorporate this (happy also if you want to reword it).

>Should we also mention 3.5 support?

FWIW my feeling is that it probably isn't necessary (against the downside that mentioning .NET 3.5 could make Polly look outdated).  Statistically, we have only had one request around .NET3.5 support in Polly over the last 3 years.

**Review by nishanil — APPROVED** [review]
LGTM.

File: docs/architecture/microservices/implement-resilient-applications/implement-http-call-retries-exponential-backoff-polly.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ ms.date: 01/07/2019
 
 The recommended approach for retries with exponential backoff is to take advantage of more advanced .NET libraries like the open-source [Polly library](https://github.com/App-vNext/Polly).
 
-Polly is a .NET library that provides resilience and transient-fault handling capabilities. You can implement those capabilities by applying Polly policies such as Retry, Circuit Breaker, Bulkhead Isolation, Timeout, and Fallback. Polly targets .NET 4.x and the .NET Standard Library 1.0 (which supports .NET Core).
+Polly is a .NET library that provides resilience and transient-fault handling capabilities. You can implement those capabilities by applying Polly policies such as Retry, Circuit Breaker, Bulkhead Isolation, Timeout, and Fallback. Polly targets .NET Framework 4.x and the .NET Standard 1.0, 1.1 and 2.0 (which supports .NET Core).
```
Comment (mairaw) [inline docs/architecture/microservices/implement-resilient-applications/implement-http-call-retries-exponential-backoff-polly.md]:
```suggestion
Polly is a .NET library that provides resilience and transient-fault handling capabilities. You can implement those capabilities by applying Polly policies such as Retry, Circuit Breaker, Bulkhead Isolation, Timeout, and Fallback. Polly targets .NET Framework 4.x and .NET Standard 1.0, 1.1, and 2.0, which supports .NET Core.
```

**Comment by mairaw** [comment]
Looks good @reisenberger. Made just one more suggestion.

**Comment by reisenberger** [comment]
Ah, the Oxford comma! Thanks for picking that up, committed. Thanks again @mairaw @nishanil  for taking the time to review and accept this.

**Review by mairaw — APPROVED** [review]
Thank you @reisenberger!

**Pull request merged**